app/_locales/en/messages.json

```diff
@@ -1782,6 +1782,9 @@
   "swapAggregator": {
     "message": "Aggregator"
   },
+  "swapAllowSwappingOf": {
+    "message": "Allow swapping of $1"
```

Comment:
For any messages where we use `$1`, `$2`, etc., we're requiring a `description` with it to ensure the localizer knows what each of them represents.

Reply:
Great to know, will add it!